core/db/src/main/resources/xsds/opennms-datasources.xsd

```diff
@@ -30,19 +30,19 @@
       <attribute name="factory" type="string" use="optional" default="org.opennms.core.db.HikariCPConnectionFactory">
         <annotation><documentation>The connection pool implementation to use.</documentation></annotation>
       </attribute>
-      <attribute name="idleTimeout" type="int" use="optional" default="600">
+      <attribute name="idleTimeout" type="string" use="optional" default="600">
```

Comment:
No blocker to me, just out of curiosity, is there a reason we can't validate against a more strict integer instead of a string? It would prevent us to catch errors in case a user writes something else than an integer in this field, isn't it?